hw_2/src/main/java/com/epam/spring/homework2/beans/AnotherBean.java

```diff
@@ -0,0 +1,18 @@
+package com.epam.spring.homework2.beans;
+
+import com.epam.spring.homework2.validator.MyValidator;
+import org.springframework.beans.BeansException;
+import org.springframework.beans.factory.config.BeanPostProcessor;
+import org.springframework.stereotype.Component;
+
+@Component
+public class AnotherBean implements BeanPostProcessor {
```

Comment:
Names for  post processors should end with `...BeanPostProcessor` in this case good name would be `ValidationBeanPostProcessor` since it performs validation

Reply:
fixed